### ISSUE TITLE
`JSJoda.Instant is not assignable to parameter of type 'import (.../core/dist/js-joda).Instant`

### ISSUE THREAD

**Issue by mg-vmoaero** [js-joda/js-joda#347]
the new published extra package seems to be missing Instant definitions from core, so its unusable.

**Comment by pithu** [comment]
Hello @mg-vmoaero 

thx for reporting and using js-joda. The typescript definition are  community driven, if you explain your issue in more  detail you might get some help here.

I would guess that your issue is, that you import the core instances like Interval from the extra package and not from core package.

**Comment by mg-vmoaero** [comment]
I'm confused about your answer. Interval only exists in the extra package, it's impossible to import it from the core package. I believe the issue is that the type definitions for Interval in extra contain references to an 'Instance' property of 'JSJoda' that contains no properties rather than a reference to the real instant from js-joda core.

**Comment by pithu** [comment]
Look like the typscript definition of the extra package is incomplete or wrong. @brandonbloom any ideas what is going wrong there.

**Comment by brandonbloom** [comment]
In my local typings directory, where I pulled these types out of, I have:

```typescript
declare module '@js-joda/extra' {
  import { Duration, Instant } from '@js-joda/core';
  ...
```

instead of:

```typescript
declare module JSJoda {
  class Duration {}
  class Instant {}
}

declare namespace JSJodaExtra {
  import Duration = JSJoda.Duration;
  import Instant = JSJoda.Instant;
  ...
```

When you asked for tests, I could not get the former to work in the context of a non-ts project. I'm not an expert in .d.ts files  and am not quite sure what the differences between these two syntaxes are. I expected that the declared classes would be merged, which seems to be the behavior for interfaces and the like. Maybe there is a version mismatch in your package.json? If that's not the problem, then I'd have to defer to somebody more knowledgable about the correct way to create correlated .d.ts files.

**Comment by mg-vmoaero** [comment]
Is anyone working on fixing this to your knowledge? @pithu 

It's a problem with how the typings are declared, it's not properly combining them. Not an issue with my package.json or anything.

**Comment by pithu** [comment]
Not that I know of

**Comment by InExtremaRes** [comment]
It should be fixed by #353. With the last commits I push this compiles perfectly:

```ts
import { Interval } from '@js-joda/extra';
import { Instant, Duration, LocalDate } from '@js-joda/core';

const interval = Interval.of(Instant.now(), Duration.ofDays(2));
const date = LocalDate.ofInstant(interval.end());
```
